lib/tasks/example_data.rake

```diff
@@ -148,12 +148,80 @@ def application_form_traits_for(region)
     traits,
     traits + %i[submitted],
     (traits + %i[preliminary_check] if region.requires_preliminary_check),
-    traits + %i[waiting_on],
+    traits + %i[submitted action_required_by_external],
+    traits + %i[submitted action_required_by_admin],
     traits + %i[awarded],
     traits + %i[declined],
   ].compact
 end
 
+def create_requestables(application_form, assessment, state)
+  status_prefix = {
+    expired: "overdue",
+    received: "received",
+    requested: "waiting_on",
+  }.fetch(state)
+
+  unless application_form.teaching_authority_provides_written_statement
+    assessment.sections.update_all(passed: true)
+  end
+
+  if application_form.teaching_authority_provides_written_statement ||
+       (
+         application_form.needs_written_statement && state != :received &&
+           rand(4).zero?
```

Comment:
This so that when we're generating example data for the review apps, we get a random variety of different example applications, it's a bit crude but it prevents us ending up with loads of applications that have LoPS and then almost none with references or qualifications.